### DISCUSSION TITLE
See MFA status on User page, and setting to require MFA for all users

### DISCUSSION THREAD

**Discussion by yaroz** [rustdesk/rustdesk-server-pro#368] [Ideas]
Since we can't lock down port 21114 for accessing the webpage, I want to ensure that all users are using MFA.  On the user,  page, I'd like to see a column that identifies if the user has set up MFA and ultimately, I'd like to see a setting that requires all users to set up MFA.

**Comment by rustdesk** [comment]
Make sense, we will do indeed, but this is not in high priority yet.

**Comment by CryptoMears** [comment]
Enforcing users that have an account to have 2FA like TeamViewer has would be very beneficial and also give your Web Console paid-for version more great advertising of your product to potential customers. You can show them they also can enforce user's accounts to have 2FA enabled which will provide better security and lower the risks of users gaining unauthorized access.

**Comment by Binary-Bandit64** [comment]
This is necessary to enable organisations that audit their security to use RustDesk.

**Comment by rustdesk** [comment]
2FA enforce is ready now, https://github.com/rustdesk/rustdesk-server-pro/releases/tag/1.6.7
